README.md

```diff
@@ -240,6 +240,28 @@ apollo {
 }
 ```
 
+### Support File Upload
+
+Apollo supports file uploading over [graphql-multipart-request-spec](https://github.com/jaydenseric/graphql-multipart-request-spec).
+
+You need to define this mapping in your build.gradle file.
+
+```gradle
+apollo {
+  customTypeMapping = [
+    "Upload" : "com.apollographql.apollo.api.GraphqlUpload"
+  ]
+}
+```
+
+Then create your mutation with mimetype and a valid `File`.
+
+```java
+  mutationSingle = SingleUploadMutation.builder()
```

Comment:
I think it's better to provide the GraphQL query itself in this example:
```
mutation SingleUpload($file: Upload!) {
  singleUpload(file: $file) {
    id
    path
    filename
    mimetype
  }
}
```

Reply:
And mention that GraphQL schema should have some custom scalar type named for example `Upload `